packages/caliper-core/lib/caliper-flow.js

```diff
@@ -53,79 +112,99 @@ module.exports.run = async function(absConfigFile, absNetworkFile, admin, client
     let configObject = CaliperUtils.parseYaml(absConfigFile);
     let networkObject = CaliperUtils.parseYaml(absNetworkFile);
 
-    let skipStart = Config.get(Config.keys.SkipStartScript, false);
-    let skipEnd = Config.get(Config.keys.SkipEndScript, false);
-
     try {
-        if (networkObject.hasOwnProperty('caliper') && networkObject.caliper.hasOwnProperty('command') && networkObject.caliper.command.hasOwnProperty('start')) {
-            if (!networkObject.caliper.command.start.trim()) {
-                throw new Error('Start command is specified but it is empty');
-            }
-            if(!skipStart){
-                const cmd = 'cd ' + workspace + ';' + networkObject.caliper.command.start;
-                await CaliperUtils.execAsync(cmd);
+        // Conditional running of 'start' commands
+        if (!performStart)  {
+            logger.info('Skipping start commands due to benchmark flow conditioning');
+        } else {
+            if (networkObject.hasOwnProperty('caliper') && networkObject.caliper.hasOwnProperty('command') && networkObject.caliper.command.hasOwnProperty('start')) {
+                if (!networkObject.caliper.command.start.trim()) {
+                    throw new Error('Start command is specified but it is empty');
+                } else {
+                    const cmd = 'cd ' + workspace + ';' + networkObject.caliper.command.start;
+                    await CaliperUtils.execAsync(cmd);
+                }
             }
         }
 
-        await adminClient.init();
-        await adminClient.installSmartContract();
-        let numberOfClients = await clientOrchestrator.init();
-        let clientArgs = await adminClient.prepareClients(numberOfClients);
+        // Conditional network initialisation
+        if (!performInit) {
```

Comment:
Init also includes the creation of connection profiles, for example (or other general init steps for the adapter objects). Can we really skip this part?